src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -409,14 +409,33 @@ public string SelectedPackagePathForInstall
         /// Settings that apply to view extensions.
         /// </summary>
         public List<ViewExtensionSettings> ViewExtensionSettings { get; set; }
+
+        private bool disableBuiltinPackages;
         /// <summary>
         /// If enabled Dynamo Built-In Packages will not be loaded.
         /// </summary>
-        public bool DisableBuiltinPackages { get; set; }
+        public bool DisableBuiltinPackages { 
+            get { return disableBuiltinPackages; }
+            set 
+            {
+                disableBuiltinPackages = value;
+                RaisePropertyChanged(nameof(DisableBuiltinPackages)); 
+            } 
+        }
+
+        private bool disableCustomPackageLocations;
         /// <summary>
         /// If enabled user's custom package locations will not be loaded.
         /// </summary>
-        public bool DisableCustomPackageLocations { get; set; }
+        public bool DisableCustomPackageLocations 
+        { 
+            get { return disableCustomPackageLocations; } 
+            set 
+            { 
+                disableCustomPackageLocations = value;
+                RaisePropertyChanged(nameof(DisableCustomPackageLocations));
+            } 
+        }
```

Comment:
I wonder if this is a good opportunity to add an array of package paths to skip loading...instead of 2 booleans.
Similar to the `PackageDirectoriesToUninstall`.
I know we talked about this before, that it would complicate the loading logic.
But looks like it will get complicated no matter what...